src/utils.ts

```diff
@@ -2,11 +2,11 @@ import AndroidUiautomator2Driver from 'appium-uiautomator2-driver';
 import XCUITestDriver from 'appium-xcuitest-driver/build/lib/driver';
 import { log } from './logger';
 import { findAPortNotInUse } from 'portscanner';
+import { waitForCondition } from 'asyncbox';
+import { JWProxy } from '@appium/base-driver';
 
-const FLUTTER_SERVER_START_MESSAGE = new RegExp(
-  /Appium flutter server is listening on port (\d+)/,
-);
-const DEVICE_PORT_RANGE = [9000, 9299];
+const DEVICE_PORT_RANGE = [9000, 9020];
+const SYSTEM_PORT_RANGE = [8000, 8100];
```

Comment:
I would rather select some ranges to not be in conflict with existing xcuitest and uia2 ranges.

How about 11000 - 11100?

Reply:
Sounds good.